twitter-covid/twitter-covid-us/build.sbt

```diff
@@ -1,19 +1,15 @@
-name := "HashtagByRegion"
+scalaVersion := "2.12.10"
 
-version := "0.1"
+name := "COVID-ANALYSIS-1"
+organization := "com.revature.michael"
```

Comment:
might just want "com.revature" and have your name in the contributors in the readme